### PR TITLE
Task/pas 22 login screen

### PR DESCRIPTION
**Pull request by lukszymanski**

### CODE REVIEW

File: libs/auth/sign-in/src/lib/components/google-button-login/types.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react';
```
Comment (pawelkondraciuk) [inline libs/auth/sign-in/src/lib/components/google-button-login/types.ts:1]: Types has been updated, see code on the master

https://github.com/instrybutor/pasnik/blob/master/apps/frontend/src/app/GoogleButtonLogin/types.ts

File: libs/auth/sign-in/src/lib/components/google-one-tap-login/types.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react';
```
Comment (pawelkondraciuk) [inline libs/auth/sign-in/src/lib/components/google-one-tap-login/types.ts:1]: Same here

**Review by pawelkondraciuk — CHANGES_REQUESTED** [review]

File: libs/auth/sign-in/src/lib/components/google-one-tap-login/types.ts (hunk 2 above)
Comment (lukszymanski) [reply libs/auth/sign-in/src/lib/components/google-one-tap-login/types.ts]: fixed

File: libs/auth/sign-in/src/lib/components/google-button-login/types.ts (hunk 1 above)
Comment (lukszymanski) [reply libs/auth/sign-in/src/lib/components/google-button-login/types.ts:1]: fixed

**Comment by pawelkondraciuk** [comment]
[Test](https://trello.com/c/j5VbSp6W/1-test)

**Review by pawelkondraciuk — APPROVED** [review]

**Comment by pawelkondraciuk** [comment]
[Database structure (UML)](https://trello.com/c/dxpnQdvS/7-database-structure-uml)

**Pull request merged**